libc/src/__support/CPP/atomic.h

```diff
@@ -71,7 +71,7 @@ template <typename T> struct Atomic {
 
   T load(MemoryOrder mem_ord = MemoryOrder::SEQ_CST,
          [[maybe_unused]] MemoryScope mem_scope = MemoryScope::DEVICE) {
-    if constexpr (LIBC_HAS_BUILTIN(__scoped_atomic_load_n))
+    if constexpr (__has_builtin(__scoped_atomic_load_n))
```

Comment:
Thanks!

While you're here, do you mind converting this code from using

`if constexpr(__has_builtin(foo))`
to
`#if __has_builtin(foo)`
?

While it may not be as pretty (requires an `#endif`), it's better for compile time to evaluate this in the preprocessor than during constexpr evaluation.  Both sides of the condition need to be lexed+parsed as written.  Perhaps not meaningfully faster though.

Reply:
Done.

Reply:
Mind doing this throughout the rest of your changes to libc/src/__support/CPP/atomic.h, too?

Reply:
Ups! Missed those, sorry.